pkg/api/pod/warnings.go

```diff
@@ -252,6 +252,17 @@ func warningsForPodSpecAndMeta(fieldPath *field.Path, podSpec *api.PodSpec, meta
 				}
 			}
 		}
+		// warning on duplicate containers[*].ports only for pod update
+		if len(c.Ports) > 1 && oldPodSpec != nil {
+			ports := sets.NewInt64()
+			for i, port := range c.Ports {
+				if ports.Has(int64(port.ContainerPort)) {
+					warnings = append(warnings, fmt.Sprintf("%s: duplicate container port %d", p.Child("ports").Index(i).Child("containerPort"), port.ContainerPort))
```

Comment:
nitpick
```suggestion
					warnings = append(warnings, fmt.Sprintf("%s: duplicated container port %d", p.Child("ports").Index(i).Child("containerPort"), port.ContainerPort))
```

Reply:
> Duplicate Data: Entries that have been added by a system user multiple times, for example, re-registering because you have forgotten your details.
> 
> Duplicated Data: Someone has deliberately taken a precise duplicate of the data - or a proportion of it - maybe for backup or reporting purposes. It may have been accidentally added to the original.

https://english.stackexchange.com/a/67464. According to this, `duplicate` is more accurate.
BTW, other duplicate warning are using `duplicate`.